kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -123,12 +125,9 @@ private[kyuubi] class EngineRef private(conf: KyuubiConf, user: String, sessionI
       }
   }
 
-  private def get(zkClient: CuratorFramework): Option[(String, Int)] = {
-    getServerHost(zkClient, engineSpace)
-  }
-
   private def create(zkClient: CuratorFramework): (String, Int) = tryWithLock(zkClient) {
-    var engineRef = get(zkClient)
+    // TODO: improve this after support engine pool
```

Comment:
can we also add the issue id？

Reply:
issue id? ❓

Reply:
you mean add https://github.com/apache/incubator-kyuubi/issues/918 ?